### DISCUSSION TITLE
用无法购买类型制作的配置文件正式开票后可以直接用吗

### DISCUSSION THREAD

**Discussion by meccimiaowu** [mikumifa/biliTickerBuy#198] [提问专区]
如题
![image](https://github.com/mikumifa/biliTickerBuy/assets/154745776/632bf64e-abf0-4c60-ac0c-38d3f36b1df2)

**Comment by meccimiaowu** [comment]
传错图了，应该是这个
![af7c52c2-116f-478c-8493-4407fa66dc3b](https://github.com/mikumifa/biliTickerBuy/assets/154745776/2657dafc-c948-48a0-88b4-078d7283670d)

**Comment by mikumifa** [comment]
可以的，这个就是今年的票
